packages/orchestration/src/orchestration-stream-chunk-response.test.ts

```diff
@@ -5,11 +5,13 @@ describe('Orchestration chat completion stream chunk response', () => {
   let mockResponses: {
     tokenUsageAndFinishReasonResponse: any;
     deltaContentResponse: any;
+    toolCallResponse: any;
```

Comment:
```suggestion
    deltaToolCallResponse: any;
```

Reply:
Since we named the other one deltaContentResponse, I would keep it aligned (also clarifies that only the delta is tested)